pundit-matchers.gemspec

```diff
@@ -12,8 +12,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.homepage    = 'https://github.com/punditcommunity/pundit-matchers'
   s.license     = 'MIT'
-  s.add_runtime_dependency 'rspec-rails', '>= 3.0.0'
```

Comment:
I do not understand why `rspec-rails` was a requirement, if this is only using `rspec`